### ISSUE TITLE
Coloured text for LPhy script

### ISSUE THREAD

**Issue by walterxie** [LinguaPhylo/linguaPhylo.github.io#2]
One idea is to convert RTF into HTML:

But Linux seems to have no better solution at the moment. I tried `unrtf --html data.rtf > data.html`, but it couldn't keep the indents somehow.

**Comment by walterxie** [comment]
UnRTF does not work with white spaces. So I fixed Java code in LinguaPhylo/linguaPhylo#119.

I attempted to use [Liquid](https://shopify.github.io/liquid/) to split lphy script html into data and model parts, but it was very difficult to identify a proper separator. "mdoel" was used, but it had to be `prepend` with hard coded style after `split`, because the style was broken. 

It'd better to change the context and show the whole script at one time
